### PR TITLE
Documentation/prepare readthedocs

### LINKED ISSUE #94: Cross-linking between datasets in docs doesn't work
See 1. in https://github.com/rl-institut-private/digipipe/pull/92#pullrequestreview-1450716008

### PR DESCRIPTION
**Pull request by dvdstrzl**
Fixes #94  and adds necessary files for successfully building the docs via readthedocs.

## Before merging into `dev`-branch, please make sure that the following points are checked:

- [X] All pre-commit tests passed locally with: `pre-commit run -a`
- [X] File `CHANGELOG.md` was updated
~- [ ] The docs were updated~
  ~- [ ] if `dataset.md`s were updated, the dataset docs were regenerated using
    `python docs/generate_dataset_mds.py`~

If packages were modified:
- [X] File `poetry.lock` was updated with: `poetry lock`
- [X] A new env was successfully set up

### CODE REVIEW

File: docs/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+mkdocs == 1.4.3
+mkdocs-material == 9.1.12
+mkdocs-include-markdown-plugin == 4.0.4
+mkdocs-same-dir == 0.1.2
+mkdocs-exclude == 1.0.2
+mkdocs_exclude_search == 0.6.5
```
Comment (nesnoj) [inline docs/requirements.txt:6]: Is there a reason why you pinned the versions? If not and it's working without pinning, please remove

**Review by nesnoj — COMMENTED** [review]
Thx @dvdstrzl for the deployment :)
Just one small question below

File: docs/requirements.txt (hunk 1 above)
Comment (dvdstrzl) [reply docs/requirements.txt]: Alright, seems to be working without the pinning just as well. :)

**Review by nesnoj — APPROVED** [review]
Great, thx! :D

**Pull request merged**